source/frontend/src/components/layout/Header/Header.tsx

```diff
@@ -71,24 +49,11 @@ export const Header = () => {
         <VerticalBar />
       </div>
       {keycloak.obj.authenticated && <UserProfile />}
-      <Nav className="other">
-        {errors && errors.length > 0 ? (
-          <FaBomb size={30} className="errors" onClick={handleShow} />
-        ) : null}
-      </Nav>
-      <ErrorModal errors={errors} show={show} setShow={setShow}></ErrorModal>
+      <div className="other"></div>
```

Comment:
This should probably be removed entirely if unused.

Reply:
I inserted that div to keep the same dimensions on the header. Without it, the user's name goes too much to the right. Let me know your thought, otherwise I can insert the css from "other" to the parent element?